src/hooks/useOffers.ts

```diff
@@ -52,6 +51,20 @@ const useOffers = () => {
       offersDict,
     };
   }, [allOffers, isLoading, hasPremierOffer, offerType]);
+
+  return checkoutService
+    ? offerData
+    : {
+        hasTVODOffers: false,
+        hasMultipleOfferTypes: false,
+        isLoading: false,
+        hasPremierOffer: false,
+        defaultOfferId: '',
+        offerType: 'svod' as OfferType,
+        setOfferType: () => null,
+        offers: [],
+        offersDict: {},
+      };
```

Comment:
This change feels a bit hacky and is needed because the `useOffers` hook is used for AVOD and AUTHVOD platforms as well. Can't we better prevent using this hook instead of returning default data?